### PR TITLE
Cleanup auth

### PR DESCRIPTION
**Pull request by davidedmundson**

### CODE REVIEW

**Comment by plfiorini** [comment]
lgtm, i was busy with the holidays and forgot to reply to your email
just one comment

File: src/daemon/Display.cpp [hunk 1]
```diff
@@ -264,6 +264,11 @@ namespace SDDM {
     }
 
     void Display::startAuth(const QString &user, const QString &password, const Session &session) {
+
+        if (m_auth->isActive()) {
+            return;
```
Comment (plfiorini) [inline src/daemon/Display.cpp]: maybe we should put a warning before return to tell auth was previously done?

**Comment by plfiorini** [comment]
ops i just realized that this got merged in `master` rather than `develop`

**Comment by davidedmundson** [comment]
Yes, I wanted it in stable?

**Comment by plfiorini** [comment]
`develop` is branched for new releases, master is not. Unless it's a hotfix for the previous release and in that case we should cherry pick from `develop`

**Pull request merged**